_data/other.yml

```diff
@@ -261,6 +261,13 @@ websites:
       exceptions:
           text: "2FA only for North American customers that create Personal campaigns."
 
+    - name: Hydrawise
+      url: https://www.hydrawise.com/
+      img: hydrawise.png
+      tfa: No
+      tritter: Hunter_Ind
```

Comment:
This should be `twitter`, not tritter. didn't notice it the first time, but spelling is important ;)